### ISSUE TITLE
DYNAMODB_SHARE_DB environment variable not honored by LocalStackContainer java client

### ISSUE THREAD

**Issue by giriprashant7891** [testcontainers/testcontainers-java#5394]
Even if I set the DYNAMODB_SHARE_DB parameter in the LocalStackContainer java client, it is still not honoring it.

```
public static final LocalStackContainer LOCALSTACK_CONTAINER =
            new LocalStackContainer(localstackImage)
                    .withServices(Service.DYNAMODB)
                    .withEnv("DEBUG", "1")
                    .withEnv("DYNAMODB_SHARE_DB", "1")
                    .withImagePullPolicy(PullPolicy.alwaysPull());
```

The logged output still looks like below:

```
org.testcontainers.containers.output.WaitingConsumer - STDOUT: 
org.testcontainers.containers.output.WaitingConsumer - STDOUT: CorsParams:	*
org.testcontainers.containers.output.WaitingConsumer - STDOUT: shouldDelayTransientStatuses:	false
org.testcontainers.containers.output.WaitingConsumer - STDOUT: SharedDb:	false
org.testcontainers.containers.output.WaitingConsumer - STDOUT: DbPath:	null
org.testcontainers.containers.output.WaitingConsumer - STDOUT: InMemory:	true
org.testcontainers.containers.output.WaitingConsumer - STDOUT: Port:	43503
org.testcontainers.containers.output.WaitingConsumer - STDOUT: Initializing DynamoDB Local with the following configuration:
org.testcontainers.containers.output.WaitingConsumer - STDOUT: Listening at http://:::42875
```

I was expecting that the DynamoDB container that comes up should have a shared DB. I'm running the `1.17.2` version of TestContainers.

**Comment by eddumelendez** [comment]
which localstack version are you using? AFAIK, this is available in 0.14.3. see [localstack's changelog](https://github.com/localstack/localstack/blob/master/CHANGELOG.md)

**Comment by giriprashant7891** [comment]
@eddumelendez  Unfortunately, I can only use the versions present in my organizations repository at the time being (which is 0.13.0). But it is no worry, for the time being, I have figured out a way around this. If this issue has been fixed in later versions, I'll close out this ticket.
